src/SDKs/RedisCache/Management.Redis/Properties/AssemblyInfo.cs

```diff
@@ -5,15 +5,15 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Azure Redis Cache Management Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure redis cache management operations.")]
+//[assembly: AssemblyTitle("Microsoft Azure Redis Cache Management Library")]
+//[assembly: AssemblyDescription("Provides Microsoft Azure redis cache management operations.")]
 
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.3.0.0")]
+//[assembly: AssemblyVersion("4.0.0.0")]
+//[assembly: AssemblyFileVersion("4.3.0.0")]
```

Comment:
@SiddharthChatrolaMs any reason these attributes are commented.
If you have not pulled latest from upstream, please do and also run the following from the root
msbuild build.proj (this will pull the latest build tools)
then do
msbuild build.proj /t:Build /p:Scope=SDKs\RedisCache
this will ensure two things:
1) the duplicate attribute errors will be gone
2) will update the package meta data for API version for your nuget package.